### PR TITLE
Add text_log.level to limit entries that goes to system.text_log table

### PR DESCRIPTION
**Pull request by azat**
Changelog category (leave one):
- Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add text_log.level to limit entries that goes to system.text_log table

### CODE REVIEW

File: dbms/programs/server/Server.cpp [hunk 1]
```diff
@@ -864,7 +864,11 @@ int Server::main(const std::vector<std::string> & /*args*/)
         for (auto & server : servers)
             server->start();
 
-        setTextLog(global_context->getTextLog());
+        {
+            String level_str = config().getString("text_log.level", "");
+            int level = level_str.empty() ? 0 : Poco::Logger::parseLevel(level_str);
```
Comment (alexey-milovidov) [inline dbms/programs/server/Server.cpp]: Ok, but zero is "none" and we expect maximum logging to text log by default.

Comment (azat) [reply dbms/programs/server/Server.cpp]: Ok, thanks! (I was looking here and did not find that zero is used, but obviously I had not though about none )

**Comment by azat** [comment]
Tests has not been restarted after the push, @alexey-milovidov does CI works?

P.S. AFAIK CI requires `can be tested`, but this PR has not this label, something has been changed?

**Comment by alexey-milovidov** [comment]
Yes, we've added you to the list for auto-launch.

**Comment by azat** [comment]
Style check is not from #8808 I guess

**Pull request merged**